docs/index.md

```diff
@@ -1,31 +1,23 @@
 Zanata Server
 =============
+Please visit [our about page](http://zanata.org/about/)
 
-TBD
-
-Features
---------
```

Comment:
The feature section should be in http://zanata.org/features/. 
I think its best if we can focus on zanata-admin(self host) documentation in readthedocs and maintainer/users guide in zanata.org/help.